package.json

```diff
@@ -26,32 +26,32 @@
     }
   },
   "dependencies": {
-    "assertive": "^2.1.1",
-    "debug": "^2.2.0",
+    "assertive": "^3.0.0",
+    "debug": "^4.1.1",
     "gofer": "^3.5.8",
-    "lighthouse": "^3.0.1",
-    "lodash": "^4.6.1",
-    "puppeteer": "^1.5.0",
-    "testium-cookie": "^1.0.0",
-    "uuid": "^3.1.0",
-    "wd": "^1.10.1"
+    "lighthouse": "^4.1.0",
```

Comment:
ditto lighthouse major bump

Reply:
v4.0.0 updates to the new scoring system that was introduced in 2018. If lighthouse is exposed through testium, any deviation to the complex configuration documentation on the lighthouse repo should be avoided.